metadata/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "subxt-metadata"
```

Comment:
Why add a new CLI rather than a command in with the existing `subxt-cli`?

Reply:
Extending the `subxt-cli` makes a lot more sense. I was initially thinking of getting the CLI out for review, will move it to the existing cli :D Thanks for review

Reply:
The common hashing functionality of the `subxt-metadata` is needed by `codegen` and `subxt` for static validation of metadata. It seems better to have it exist on its own for now, to avoid introducing a dependency on codegen in subxt (as previously). What do you think? 😄

Reply:
Ok I see, fine to keep this as its own lib crate if it is shared, but still move the CLI stuff to `subxt-cli`?

Reply:
Yep exactly, keep the lib crate and move the CLI functionality in `subxt-cli` :D Thanks